README.md

```diff
@@ -1 +1,89 @@
-# cnp-module-automation-runbook-app-secret-alert
\ No newline at end of file
+## App Registration Alert Module
+
+This Module will create a PowerShell runbook which checks for App Registration Client Secrets that are expiring within 21 days.
+
+```terraform
+module "app_registration_alert" {
+  source = "../"
```

Comment:
change to `source = "git::https://github.com/hmcts/cnp-module-automation-runbook-app-secret-alert"`